sysutils/freeradius/Portfile

```diff
@@ -1,22 +1,26 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
-PortGroup               muniversal 1.0
-
+PortGroup               github      1.0
+PortGroup               muniversal  1.0
+PortGroup               openssl     1.0
+PortGroup               perl5       1.0
+
+github.setup            FreeRADIUS freeradius-server 3.0.25
+git.branch              release_[string map {. _} ${github.version}]
+github.tarball_from     releases
 name                    freeradius
-version                 3.0.21
-revision                5
-checksums               rmd160  04a038b701f19d9c598e826a795a0cdaacd3768b \
-                        sha256  c22dad43954b0cbc957564d3f8cbb942ff09853852d2c2155d54e6bd641a4e7d \
-                        size    3184588
+revision                0
+checksums               rmd160  d377302aaac14eda511d5cd706080382820bdbc2 \
+                        sha256  4883a68bb3cee46025359a97c1691ce655e8babfd6dc3c021d00e109a2ff4c10 \
+                        size    5299624
 
 categories              sysutils
 maintainers             nomaintainer
 homepage                https://freeradius.org
 platforms               darwin freebsd
 license                 LGPL-2.1+
-use_bzip2               yes
-distname                ${name}-server-${version}
+distname                ${name}-server-${github.version}
```

Comment:
```suggestion
distname                ${name}-server-${version}
```

Reply:
Also, why remove `use_bzip2 yes`?

Reply:
I had originally removed `use_bzip2` when switching to using releases, but have just restored it.